armory/instrument/config.py

```diff
@@ -253,6 +255,11 @@ def _task_metric(
     elif name == "word_error_rate":
         final = metrics.get("total_wer")
         final_suffix = "total_word_error_rate"
+    elif name == "per_class_mean_accuracy":
+        metric = metrics.get("identity_unzip")
+        func = metrics.get("per_class_mean_accuracy")
+        final = lambda x: func(*metrics.task.identity_zip(x))
```

Comment:
flake8 didn't like that I used a lambda here.  It told me to "use a def".  Is that really better for this situation?

Reply:
A lambda one-liner is a bit odd with assignment. It's essentially the same as:
```
def final(x):
    return func(*metrics.task.identity_zip(x))
```
I don't see any issues with using that.